cypress/integration/brewery/TableParameters-fullscreen.spec.js

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { Scenarios, ScenarioParameters, TableParameters, Login } from '../../commons/actions';
+import { BreweryParameters } from '../../commons/actions/brewery';
+import { stub } from '../../commons/services/stubbing';
+import { SOLUTIONS } from '../../fixtures/stubbing/TableParameters-fullscreen/solutions';
+
+const getFullscreenTable = TableParameters.getFullscreenTable;
+
+const getVipCustomersTable = () => {
+  return cy.get('[data-cy=table-vipCustomers]');
+};
+
+describe('check every TableToolbar component', () => {
+  before(() => {
+    stub.start({
+      GET_DATASETS: true,
+      GET_SCENARIOS: true,
+      GET_WORKSPACES: true,
+      GET_SOLUTIONS: true,
+      PERMISSIONS_MAPPING: true,
+    });
+    stub.setSolutions(SOLUTIONS);
+  });
+
+  beforeEach(() => {
+    Login.login();
+  });
+
+  after(() => {
+    stub.stop();
+  });
+
+  it('Edit a cell, and check if value as change in and out of fullscreen', () => {
+    Scenarios.getScenarioViewTab(60).should('be.visible');
+    ScenarioParameters.expandParametersAccordion();
+    BreweryParameters.switchToCustomersTab();
+    BreweryParameters.getCustomersTable().should('be.visible');
+    BreweryParameters.getCustomersTableLabel().should('be.visible').should('have.text', 'Customers');
+    BreweryParameters.getCustomersTableGrid().should('exist');
+    BreweryParameters.getCustomersFullscreenButton().should('not.exist');
+    BreweryParameters.importCustomersTableData('customers.csv');
+    TableParameters.importFile(getVipCustomersTable(), 'customers.csv');
+
+    BreweryParameters.getCustomersFullscreenButton().should('be.visible');
+    BreweryParameters.enterCustomersFullscreenButton();
+    TableParameters.editStringCell(getFullscreenTable, 'age', 0, '11', true).should('have.text', '11');
+    TableParameters.editStringCell(getFullscreenTable, 'canDrinkAlcohol', 0, 'true', true).should('have.text', 'true');
+    TableParameters.editStringCell(getFullscreenTable, 'favoriteDrink', 0, 'Wine', true).should('have.text', 'Wine');
+    TableParameters.editStringCell(getFullscreenTable, 'birthday', 0, '02/05/2012', true).should(
+      'have.text',
+      '02/05/2012'
+    );
+    BreweryParameters.exitCustomersFullscreenButton();
+
+    BreweryParameters.getCustomersTableCell('age', 0).should('have.text', '11');
+    BreweryParameters.getCustomersTableCell('canDrinkAlcohol', 0).should('have.text', 'true');
+    BreweryParameters.getCustomersTableCell('favoriteDrink', 0).should('have.text', 'Wine');
+    BreweryParameters.getCustomersTableCell('birthday', 0).should('have.text', '02/05/2012');
+
+    TableParameters.getCell(getVipCustomersTable(), 'vipLevel', 0).should('have.text', '10');
```

Comment:
I can't find where this value "10" comes from, but maybe your answer to my previous comment will answer this question too :slightly_smiling_face:

Reply:
Default value of file (check if the modification in customer table do not affect customervip one)